assets/css/stripeStyle.css

```diff
@@ -127,8 +109,159 @@ input:focus {
 
 .form-field-container {
 	margin: 10px 0 20px 0;
+	width: 50%;
+	padding: 0.5rem;
 }
 
 .hidden {
 	display: none;
 }
+
+form {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 0.5rem;
+}
+
+form h2 {
+	width: 100%;
+	padding: 0.5rem;
+	color: #304ffe;
+}
+
+.form-field-container label {
+	font: 600 12px/17px "Open Sans", sans-serif;
+	color: #616161;
+}
+
+.form-field-container input,
+.form-field-container select {
+	margin-top: 1rem;
+}
+
+.form-field-container input.prefix-input {
+	margin-top: 0;
+}
+
+.form-field-container.card {
+	width: 100%;
+}
+
+.form-field-container.card label {
+	width: 100%;
+	float: left;
+}
+
+.form-field-container.card .num {
+	width: calc(50% - 0.5rem);
+	float: left;
+}
+
+.form-field-container.card .exp {
+	width: calc(25% - 0.25rem);
+	float: left;
+	margin-left: 1rem;
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+}
+
+.form-field-container.card .cvc {
+	width: calc(25% - 0.25rem);
+	float: right;
+	border-top-left-radius: 0;
+	border-bottom-left-radius: 0;
+}
+
+.columns {	
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5886534)